README.md

```diff
@@ -4,6 +4,11 @@
 
 Use this component to add a github gist on your website.
 
+<p><details>
```

Comment:
I'd prefer not having a huge image here if we have codesandbox examples already :)

Reply:
The `<p><details>` thing is doing magic if you have not seen it. You can check it here: https://github.com/tleunen/react-gist/blob/dbf6fb72e575bec889c71b48ec7b4f0211463c91/README.md

But otherwise, I am also open to delete it. As you wish :)

Reply:
True. I missed it :D Sounds good!

Thank you!